alomgir-alom310/week-2/week-two-project-master/shopping_page/app.js

```diff
@@ -43,12 +43,28 @@ function functionBoat() {
   }
 };
 
+$('#cars button').on('click',function () {
+  var spec = $(this).html();
+  $('#garage').append('<li>' + spec + '</li>');
```

Comment:
To clean your code I would recommend using template literals. This will increase the overall readability of your code. 
``` js 
.append(`<li>${spec}</li>`)
```